homeassistant/components/google_assistant/helpers.py

```diff
@@ -344,10 +365,13 @@ def pop_agent_user_id(self, agent_user_id):
             self._data[STORE_AGENT_USER_IDS].pop(agent_user_id, None)
             self._store.async_delay_save(lambda: self._data, 1.0)
 
-    async def async_load(self):
-        """Store current configuration to disk."""
-        if data := await self._store.async_load():
-            self._data = data
+    @callback
+    def _empty_config(self):
+        """Return an empty config."""
+        return {
+            STORE_AGENT_USER_IDS: {},
+            STORE_GOOGLE_LOCAL_WEBHOOK_ID: self._hass.components.webhook.async_generate_id(),
```

Comment:
Import `async_generate_id` and use it directly instead of going through the hass object.

Reply:
Done.